ariadne/src/Ariadne/Cardano/Knit.hs

```diff
@@ -110,7 +110,7 @@ instance ComponentDetokenizer Cardano where
     TokenAddress a -> pretty a
     TokenPublicKey pk -> sformat fullPublicKeyF pk
     TokenHash h -> sformat ("#"%hashHexF) (getAHash h)
-    TokenCoin c -> let (amount, unit) = showScientificCoin c in amount <> unit
+    TokenCoin c -> let (amount, unit) = showScientificCoin c in amount <> show unit
```

Comment:
Shouldn't there be a space between the amount and the unit?

Reply:
No, that is for parser. Knit literals for coins do not have spaces. Spaces are only for UI.